src/main/java/pl/edu/zut/mad/schedule/ScheduleController.java

```diff
@@ -43,6 +46,6 @@ public ScheduleController(GroupAlbumRepository groupAlbumRepository,
         if (studentSchedule.isEmpty()) {
             throw new NotFoundException(albumNumber);
         }
-        return studentSchedule;
+        return scheduleMapper.daysFrom(studentSchedule);
```

Comment:
I suggest to change name of method from `daysFrom` to more explanatory, maybe somenthing like `daysFromSchedule` or `toDays`. What you think?

Reply:
`daysFromSchedule` is too verbose. Method already have Schedule as argument, so adding same thing to name have opposite effect. `daysFrom(schedule)` reads like sentence, as opposed to `toDays(schedule)`, so I'd leave it as it is.

Reply:
I mean that we should know, from method name, what kind (type) of argument we should pass. Of course in that example it's not a big difference. In that specific situation I can approve that but next time we back to conversation 😄